app/styles/join-section.module.css

```diff
@@ -0,0 +1,102 @@
+.join {
+  text-align: center;
+}
+
+.join__titleHighlighted {
+  color: var(--color-pinkl);
+  margin-top: -1.1rem;
+}
+
+.join__titleQuestion {
+  font-size: 3.3rem;
+}
+
+.join__para {
+  width: 52%;
+  text-align: justify;
+  margin: auto;
+  color: var(--color-darkblack);
+  font-size: 1.4rem;
+  font-weight: bold;
+  margin-top: 13px;
+}
+
+.button {
+  display: inline-block;
+  margin-top: 2.5rem;
+  text-decoration: none;
+  color: var(--color-white);
+  font-size: 1.3rem;
```

Comment:
Can we please make the button a bit more bigger and apply a media query to make it responsive?
```CSS
font-size: 1.5rem;
border-radius: 1rem;
padding: 1.25rem 3rem;
```